Arrowgene.Ddon.GameServer/Handler/StampBonusCheckHandler.cs

```diff
@@ -10,16 +10,39 @@ public class StampBonusCheckHandler : PacketHandler<GameClient>
     {
         private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(StampBonusCheckHandler));
 
+        private readonly DdonGameServer _gameServer;
 
         public StampBonusCheckHandler(DdonGameServer server) : base(server)
         {
+            _gameServer = server;
         }
 
         public override PacketId Id => PacketId.C2S_STAMP_BONUS_CHECK_REQ;
 
         public override void Handle(GameClient client, IPacket packet)
         {
-            client.Send(InGameDump.Dump_95);
+            bool canTotal = _gameServer.StampManager.CanTotalStamp(client.Character.StampBonus);
```

Comment:
Should change this to extend `GameRequestPacketHandler` which is the newer format we are trying to use for handlers.